src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/xpath/xpathquerygenerator/InputXpathQueryGeneratorTextBlock.java

```diff
@@ -0,0 +1,10 @@
+package com.puppycrawl.tools.checkstyle.xpath.xpathquerygenerator;
+
+public class InputXpathQueryGeneratorTextBlock {
+
+    String testOne = """               // violation
+        &This is the first line
+        >This is the second line
+        <This is the third line
+        """;
+}
```

Comment:
Drone PR failing since **TextBlock** are Supported from SE 15 and non-compilable with Java 8. How to escape this test?

Reply:
Fixed.